apiserver/facades/controller/migrationtarget/migrationtarget.go

```diff
@@ -32,7 +32,7 @@ import (
 type API struct {
 	state         *state.State
 	pool          *state.StatePool
-	controllerDB  database.TxnRunner
+	controllerDB  changestream.WatchableDB
```

Comment:
Revert this back to `database.TxnRunner`.

Reply:
done